vip-helpers/vip-notoptions-mitigation.php

```diff
@@ -10,9 +10,13 @@
 
 use Automattic\VIP\Utils\Alerts;
 
+define( 'USER_ROLE_BACKUP_LENGTH', 3 );
```

Comment:
Should we prefix this with `VIP_`? And also, just curious if there would be any harm in upping this to say 5 - since we aren't autoloading it shouldn't matter, and will better help protect against maybe a weekend problem that went unnoticed for 3 days?

Reply:
@WPprodigy this file is namespaced so that'll also cover the constant.